apps/wallet/src/ui/app/pages/home/index.tsx

```diff
@@ -24,11 +24,23 @@ const HomePage = ({ disableNavigation, limitToPopUpSize = true }: Props) => {
     const initChecking = useInitializedGuard(true);
     const lockedChecking = useLockedGuard(false);
     const guardChecking = initChecking || lockedChecking;
+    const [visibility, setVisibility] = useState(
+        document?.visibilityState || null
+    );
     const dispatch = useAppDispatch();
     useEffect(() => {
-        const sub = of(guardChecking)
+        const callback = () => {
+            setVisibility(document.visibilityState);
+        };
+        document.addEventListener('visibilitychange', callback);
+        return () => {
+            document.removeEventListener('visibilitychange', callback);
+        };
+    });
```

Comment:
@pchrysochoidis Just a small nit before we release: this `useEffect` should have an empty dependency array so that it we don't re-attach event listeners every state change.

Reply:
Ops, missed it, thanks I will fix it